app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -27,6 +27,7 @@ class AuthSourceLdap < AuthSource
   include Parameterizable::ByIdName
   include Encryptable
   encrypts :account_password
+  include Taxonomix
```

Comment:
I think it would be also desirable for AuthSourceExternal, when user is authenticated externally (apache module) it would be good to assign taxonomies too